src/main/java/com/yscope/clp/compressorfrontend/NativeLibraryLoader.java

```diff
@@ -82,4 +81,8 @@ private static URL getLibUrlInJar() {
 
     return null;
   }
+
+  private static void logError(String message) {
+    System.err.println("clp-ffi: " + message);
```

Comment:
Can we print out timestamp here?

Reply:
Is this format okay?

`2023-04-09T01:18:57.823Z clp-ffi: Failed to find native library in JAR.`